metadatastore/commands.py

```diff
@@ -157,6 +157,7 @@ def insert_run_start(time, scan_id, beamline_id, beamline_config, uid=None,
     if project is None:
         project = ''
 
+    beamline_config = BeamlineConfig.objects(uid=beamline_config).first()
```

Comment:
Oh. Thanks...  I updated the notebook with ``%timeit`` results for a number of ways to search for objects in mds.  

Interesting results: 

1. ``.get()`` is slower than ``.first()`` which is slower than ``[0]``.  

1. ``RunStart.objects(__raw__={'uid':uid})`` is faster than ``RunStart.objects(uid = uid)``